README.md

```diff
@@ -75,20 +75,20 @@ This is used in conjunction with [tools supplied by Silabs][Silabs BGAPI DTM doc
-[Lyra AT Interface guide]: <https://www.lairdconnect.com/documentation/user-guide-interface-application-lyra-series>
-[Lyra AT Interface release notes]: <https://www.lairdconnect.com/documentation/release-notes-lyra-series-vx-3-1-402>
+[Lyra product brief]: <https://www.ezurio.com/documentation/product-brief-lyra-series>
+[Lyra P module datasheet]: <https://www.ezurio.com/documentation/datasheet-lyra-p>
+[Lyra S module datasheet]: <https://www.ezurio.com/documentation/datasheet-lyra-s>
+[Lyra AT Interface guide]: <https://www.ezurio.com/documentation/user-guide-interface-application-lyra-series>
+[Lyra AT Interface release notes]: <https://www.ezurio.com/documentation/release-notes-lyra-series-vx-3-1-402>
 [Bluetooth Xpress]: <https://docs.silabs.com/gecko-os/1/bgx/latest/getting-started>
-[Lyra P DVK user guide]: <https://www.lairdconnect.com/documentation/user-guide-lyra-p-development-kit>
-[Lyra S DVK user guide]: <https://www.lairdconnect.com/documentation/user-guide-lyra-s-development-kit>
-[Native C development guide]: <https://www.lairdconnect.com/documentation/user-guide-lyra-series-c-code-development>
-[Firmware options and upgrade guide]: <https://www.lairdconnect.com/documentation/user-guide-firmware-options-and-upgrading-lyra-series>
-[Lyra P BGAPI DTM Application Note]: <https://www.lairdconnect.com/documentation/application-note-using-bgapi-direct-test-mode-lyra-p>
-[Lyra S BGAPI DTM Application Note]: <https://www.lairdconnect.com/documentation/application-note-using-bgapi-direct-test-mode-lyra-s>
+[Lyra P DVK user guide]: <https://www.ezurio.com/documentation/user-guide-lyra-p-development-kit>
+[Lyra S DVK user guide]: <https://www.ezurio.com/documentation/user-guide-lyra-s-development-kit>
+[Native C development guide]: <https://www.ezurio.com/documentation/user-guide-lyra-series-c-code-development>
+[Firmware options and upgrade guide]: <https://www.ezurio.com/documentation/user-guide-firmware-options-and-upgrading-lyra-series>
+[Lyra P BGAPI DTM Application Note]: <https://www.ezurio.com/documentation/application-note-using-bgapi-direct-test-mode-lyra-p>
+[Lyra S BGAPI DTM Application Note]: <https://www.ezurio.com/documentation/application-note-using-bgapi-direct-test-mode-lyra-s>
 [Bluetooth SIG Core Specification]: <https://www.bluetooth.com/specifications/bluetooth-core-specification>
-[Bluetooth Xpress migration guide]: <https://www.lairdconnect.com/documentation/user-guide-bluetooth-xpress-bgx-migration-lyra-modules>
+[Bluetooth Xpress migration guide]: <https://www.ezurio.com/documentation/user-guide-bluetooth-xpress-bgx-migration-lyra-modules>
 [Silabs BGAPI DTM documentation]: <https://www.silabs.com/documents/public/application-notes/an1267-bt-rf-phy-evaluation-using-dtm-sdk-v3x.pdf>
 [Silabs BGAPI description]: <https://docs.silabs.com/bluetooth/3.1/bgapi>
 [Lyra Releases Page]: <https://github.com/LairdCP/Lyra_Firmware/releases/tag/GA2.2>
```

Comment:
Could update this link to GA3 and close this PR just prior to creating the release?